chart/values.schema.json

```diff
@@ -1203,6 +1216,18 @@
                             ]
                         }
                     }
+                },
+                "securityContext": {
+                    "description": "Global Pod security context as defined in `Pod Security Context <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core>`_. If not set, the values from `securityContext` will be used.",
+                    "type": "object",
```

Comment:
Can you use a more strict schema?

Reply:
Yeah, that is great, was not aware of the possibility!
Will update it accordingly.